helm/fiftyone-teams-app/README.md.gotmpl

```diff
@@ -319,6 +320,23 @@ FiftyOne Teams continues to support the use of environment variables to set
 storage credentials in the application context and is providing an alternate
 configuration path.
 
+### Static Banner Configuration
+
+Fiftyone Teams v2.5 introduces the ability to add a static banner to the
+application.
+
+Configure the Static Banner by setting the following environment variables in
+your `values.yaml`.
+
+Banner text is configured with:
+`casSettings.env.FIFTYONE_APP_BANNER_TEXT` and
+`teamsAppSettings.env.FIFTYONE_APP_BANNER_TEXT`
+
```

Comment:
good idea - @kevin-dimichel how about this

```suggestion
Examples:
- FIFTYONE_APP_BANNER_TEXT="Internal Deployment"
- FIFTYONE_APP_BANNER_COLOR="green"|"rgb(34,139,34)"|"#f1f1f1|<any-valid-color>"

```

Reply:
Awesome!  Can we please replicate (in docker parlance) this to the `docker/README.md` too?


For helm, what if we did something like?

````suggestion
```yaml
casSettings:
  env:
    FIFTYONE_APP_BANNER_COLOR: "green" # or "rgb(34,139,34)" or ""#f1f1f1"
    FIFTYONE_APP_BANNER_TEXT: "Internal Deployment"
teamsAppSettings:
  env:
    FIFTYONE_APP_BANNER_COLOR: "green" # or "rgb(34,139,34)" or ""#f1f1f1"
    FIFTYONE_APP_BANNER_TEXT: "Internal Deployment"
```
````

But down on line 339 (after we describe the background color)

Reply:
👍  captured my understanding in https://github.com/voxel51/fiftyone-teams-app-deploy/pull/279 - lmk

Reply:
#279 is approved.  Once merged in, I believe that this PR will be ready. : )